packages/graphiql/src/main.tsx

```diff
@@ -0,0 +1,188 @@
+/**
```

Comment:
yeah, this file just needs to be removed. it should instead import `renderGraphiQL.js` as before, which does not contain any typescript, and uses the UMD global